api-service/src/middlewares/setResmsgid.ts

```diff
@@ -0,0 +1,9 @@
+import { NextFunction, Request, Response } from "express";
```

Comment:
Instead of creating multiple middlewares have a common middleware for
1. API ID
2. resmsgid

may something else tomorrow

Reply:
done